.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode8
+osx_image: xcode9
```

Comment:
If you want to ensure that everything stays backwards compatible, could consider keeping both versions running on Travis.

Example from KIF: https://github.com/kif-framework/KIF/blob/master/.travis.yml

Reply:
TIL that was possible!

Though it's worth noting the move to Swift 4 will break Xcode 8. This will need to be a major version bump.